release.json

```diff
@@ -10,7 +10,7 @@
         "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
         "JMXFETCH_VERSION": "0.47.2",
         "JMXFETCH_HASH": "92f9c744fe00c13c6401829b9eb50f44c5843ae9d1a14f7618531bdf0187e44f",
-        "MACOS_BUILD_VERSION": "master",
+        "MACOS_BUILD_VERSION": "jaime/disable_coverprofile",
```

Comment:
Set back to `master` after https://github.com/DataDog/datadog-agent-macos-build/pull/130 is merged. Likewise for line 25 below.

Reply:
✅